src/classes/CON_Email_TEST.cls

```diff
@@ -106,40 +112,78 @@ private class CON_Email_TEST {
 
 	// Test the batch Class
 	@isTest static void testContactEmailBatch() {
+
+		UTIL_CustomSettingsFacade.getSettingsForTests(
+            new Hierarchy_Settings__c(Disable_Preferred_Email_Enforcement__c = false)
+            );
+
+        UTIL_CustomSettingsFacade.getSettings();
```

Comment:
Why are we calling UTIL_CustomSettingsFacade.getSettingsForTests and then UTIL_CustomSettingsFacade.getSettings()? Wouldn't the second call override the first one? Also, the Setup method is being called. Do we care about that?